<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showTransactions</class>
 <widget class="QDialog" name="showTransactions">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(192, 234, 227);
color: rgb(110,110,110);
text-transform: uppercase;</string>
  </property>
  <widget class="QLabel" name="TxtLabel_nayta_tilitapahtumat">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>561</width>
     <height>380</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>561</width>
     <height>380</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(248,248,248);
line-height: 6px;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>621</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(248,248,248);</string>
   </property>
   <widget class="QLabel" name="label_TILITAPAHTUMAT_otsikko">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 186, 111);
background-color: rgb(248,248,248);
letter-spacing: 1px;</string>
    </property>
    <property name="text">
     <string>TILITAPAHTUMAT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="nappi_sulje">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242,242,242);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>sulje ikkuna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nappi_kirjaudu_ulos">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242,242,242);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>TxtLabel_nayta_tilitapahtumat</zorder>
  <zorder>nappi_sulje</zorder>
  <zorder>nappi_kirjaudu_ulos</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
